README.md

```diff
@@ -36,47 +36,46 @@ You can install the NodeObservability Operator by building and pushing the Opera
    ```
 
 
-### Installing the `Node Observability` Operator using a custom index image on OperatorHub
-**Note**: The below procedure works best with `podman` as container engine
-    
```

Comment:
@lmzuccarelli Please share the prerequisites required to carry out this procedure.

Reply:
Done.